rust/lsp-lib/src/utils.rs

```diff
@@ -205,4 +205,11 @@ pub fn start_new_server(
     }
 
     Ok(language_server_client)
+}
+
+/// Helper Method to get Url of the View path
+/// Panics: If view does not have a path or the given path
+/// can not be converted to Url
```

Comment:
This comment doesn't seem to be accurate? As written this function doesn't panic.